### PR TITLE
Oracle edits - symbol separator change and running as Docker container

### LINKED ISSUE #88: Update Runing an AOS to add install and run as (a) a daemon service, (b) a Docker image
### Description

The how to guide for install and running AOS documents how to run and install from binary or executable.

How to run as a docker image should be added.

### Tasks

- [x] Add "How to install as a Docker image"
- [x] Add "How to run as a Docker image"
- [ ] (Optional - documented in repo README) Add "How to run as a daemon service"

### References

- #75

### LINKED ISSUE #110: Guide "Transfer Liquid Newton" - typo ' lnew' -> 'lntn'
**Describe the bug**

In https://docs.autonity.org/delegators/transfer-lntn/#get-liquid-newton-holdings, Step 1 of the Guide has a typo in the command - this should be `lntn-balances`:

```
aut account lnew-balances
{
	"0xA9F070101236476fe077F4A058C0C22E81b8A6C9": 9901
}
```

### PR DESCRIPTION
**Pull request by jay-cousins**
@raj-shekhar1 & @Jason-Zhangxin-Chen & CODEOWNER:

A PR for recent updates to the Autonity Oracle Server (AOS).

Edits to:

- Add how to install and run as a Docker container
- Editorial on how to install and run building from source
- Edit the symbol separator (from `/` to `-`) in API reference and 
- Update AOS CLI Reference to v0.1.3 tag - add new gas priority fee tip and log level flags
- Update AGC CLI Reference to v0.12.2 tag - oraclekey and oraclekeyhex flags relocated from MISC to NETWORKING OPTIONS

### CODE REVIEW

File: content/docs/oracle/run-oracle/_index.md [hunk 1]
```diff
@@ -121,16 +194,60 @@ Additional data adaptors for any external data source can be developed using the
...
 
 
-<!--
 ## Run Autonity Oracle Server as Docker image {#run-docker}
 
-TODO
--->
-<!--  
-## Run Autonity Oracle Server as Linux daemon service {#run-daemon}
+- Ensure that the Autonity Oracle Server [Docker image](/oracle/install-oracle/#install-docker) has been installed.
```
Comment (raj-shekhar1) [inline content/docs/oracle/run-oracle/_index.md]: @cmjc this sections looks duplicate else LGTM.

**Review by raj-shekhar1 — APPROVED** [review]

File: content/docs/oracle/run-oracle/_index.md (hunk 1 above)
Comment (jay-cousins) [reply content/docs/oracle/run-oracle/_index.md]: I know what you mean, but it is at the beginning of all Run sections - for AOS and for AGC, for running as docker and binary. They are hyperlinked back to the relevant Install page section. The rationale was if a user navigates straight to Run, this is a pointer straight back to how to install it in one click without further navigation...

Comment (raj-shekhar1) [reply content/docs/oracle/run-oracle/_index.md]: got it, Thanks. @cmjc

File: content/docs/reference/api/oracle/_index.md [hunk 2]
```diff
@@ -169,7 +169,11 @@ aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D getSymbol
 
 {{< tabpane langEqualsHeader=true >}}
 {{< tab header="aut" >}}
+<<<<<<< HEAD
 aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D getSymbols
+=======
+aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D getSymbols               
+>>>>>>> b68833b (Oracle, edits to add Docker run and install)
```
Comment (aiman) [inline content/docs/reference/api/oracle/_index.md]: This needs to be fixed

File: content/docs/reference/cli/oracle/_index.md [hunk 3]
```diff
@@ -32,9 +32,19 @@ Run `autoracle --help` to view the options:
...
+=======
+| `-oracle_autonity_ws_url` | WS-RPC server listening interface and port of the connected Autonity Go Client node | "ws://127.0.0.1:8546" | Yes |
+| `-oracle_gas_tip_cap` | The gas priority fee cap set for oracle data report transactions. Must be a non-zero value. | Default `1` | No |                                                             
+| `-oracle_symbols` | The currency pair symbols the oracle returns data for. A comma-separated list | "AUD-USD,CAD-USD,SEK-USD,EUR-USD,GBP-USD,JPY-USD,ATN-USD,NTN-USD" | No |
+| `-oracle_key_file` | Path to the oracle server key file | (defaults to testing key in `/test_data/keystore`) | Yes |
+| `-oracle_key_password` | Password to the oracle server key file | (defaults to password for testing key in `/test_data/keystore`) | Yes |
+| `-oracle_plugin_dir` | Path to the DIR where the data source plugins are stored | `./build/bin/plugins` | No |
+| `-oracle_plugin_conf` | Path to the data source plugins YAML configuration file `plugins-conf.yml` | `./build/bin/plugins/plugins-conf.yml` | Yes |
+>>>>>>> b68833b (Oracle, edits to add Docker run and install)
```
Comment (aiman) [inline content/docs/reference/cli/oracle/_index.md]: This needs to be fixed

**Review by aiman — CHANGES_REQUESTED** [review]

File: content/docs/reference/cli/oracle/_index.md (hunk 3 above)
Comment (jay-cousins) [reply content/docs/reference/cli/oracle/_index.md]: Fixed in commit 2246212b240621e71ca0d995cf34a15a4e61b93c.

File: content/docs/reference/api/oracle/_index.md (hunk 2 above)
Comment (jay-cousins) [reply content/docs/reference/api/oracle/_index.md]: Fixed in commit 2246212b240621e71ca0d995cf34a15a4e61b93c.

File: content/docs/reference/api/aut/op-prot/_index.md [hunk 4]
```diff
@@ -830,7 +830,11 @@ aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D setSymbol
 
 {{< tabpane langEqualsHeader=true >}}
 {{< tab header="aut" >}}
+<<<<<<< HEAD
 aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D setSymbols ["AUD/USD", "CAD/USD", "EUR/USD", "GBP/USD", "JPY/USD", "SEK/USD", "ATN/USD", "NTN/ATN"]
+=======
+aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D setSymbols ["AUD-USD", "CAD-USD", "EUR-USD", "GBP-USD", "JPY-USD", "SEK-USD", "ATN-USD", "NTN-USD"]
+>>>>>>> b68833b (Oracle, edits to add Docker run and install)
```
Comment (aiman) [inline content/docs/reference/api/aut/op-prot/_index.md]: more merge conflicts

File: content/docs/reference/api/oracle/_index.md [hunk 5]
```diff
@@ -169,7 +169,7 @@ aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D getSymbol
 
 {{< tabpane langEqualsHeader=true >}}
 {{< tab header="aut" >}}
-aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D getSymbols
+aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D getSymbols
```
Comment (aiman) [inline content/docs/reference/api/oracle/_index.md]: unnecessary whitespace

```suggestion
aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D getSymbols
```

**Review by aiman — CHANGES_REQUESTED** [review]
:bulb: Please check the diff to identify any issues, either locally or on Github before or when opening a PR.

File: content/docs/reference/api/aut/op-prot/_index.md (hunk 4 above)
Comment (jay-cousins) [reply content/docs/reference/api/aut/op-prot/_index.md]: Sorry 😞 . Fixed in commit 480540aa065b97910aa107113b7c3d671bcbd22c

**Review by aiman — APPROVED** [review]
LGTM

File: content/docs/oracle/run-oracle/_index.md [hunk 6]
```diff
@@ -34,63 +35,141 @@ Transaction costs for submitting price report data on-chain _are_ refunded but t
...
 
+
     ``` bash
     autoracle \
-        -oracle_key_file="<KEYFILE>" \
+        -oracle_key_file="<ORACLE_KEYFILE>" \
         -oracle_key_password="<PWD>" \
-        -oracle_autonity_ws_url="<WS_ADDRESS>" 
+        -oracle_autonity_ws_url="<WS_ADDRESS>"
         -oracle_plugin_conf="<PLUGINS_CONFIG_FILE>/plugins-conf.yml" ;
```
Comment (Jason-Zhangxin-Chen) [inline content/docs/oracle/run-oracle/_index.md]: The naming for flags are updated as well by according to the feedback from the game. Please update those examples in the doc by refering to: 
```shell
Usage of Autonity Oracle Server:
Sub commands: 
  version: print the version of the oracle server.
Flags:
  -key.file="./UTC--2023-02-27T09-10-19.592765887Z--b749d3d83376276ab4ddef2d9300fb5ce70ebafe": Set oracle server key file
  -key.password="123": Set the password to decrypt oracle server key file
  -log.level=2: Set the logging level, available levels are:  0: NoLevel, 1: Trace, 2:Debug, 3: Info, 4: Warn, 5: Error
  -plugin.conf="./plugins-conf.yml": Set the plugins' configuration file
  -plugin.dir="./plugins": Set the directory of the data plugins.
  -symbols="AUD-USD,CAD-USD,EUR-USD,GBP-USD,JPY-USD,SEK-USD,ATN-USD,NTN-USD,NTN-ATN": Set the symbols string separated by comma
  -tip=1: Set the gas priority fee cap to issue the oracle data report transactions.
  -ws="ws://127.0.0.1:8546": Set the WS-RPC server listening interface and port of the connected Autonity Client node
```

File: content/docs/oracle/run-oracle/_index.md [hunk 7]
```diff
@@ -34,63 +35,141 @@ Transaction costs for submitting price report data on-chain _are_ refunded but t
...
+
+   ```bash
+   docker run \
+        -t -i \
+        --volume $<ORACLE_KEYFILE>:/autoracle/oracle.key \
+        --volume $<PATH_TO_PLUGINS_FILE>:/autoracle/plugins-conf.yml \
+        --name oracle-server \
+        --rm \
+        ghcr.io/autonity/autonity-oracle:latest \
+        -oracle_key_file="/autoracle/oracle.key" \
```
Comment (Jason-Zhangxin-Chen) [inline content/docs/oracle/run-oracle/_index.md]: Update flag naming again.

File: content/docs/oracle/run-oracle/_index.md [hunk 8]
```diff
@@ -121,21 +205,60 @@ Additional data adaptors for any external data source can be developed using the
...
+
+   ```bash
+   docker run \
+        -t -i \
+        --volume $<PATH_TO_ORACLE_KEY_FILE>:/autoracle/oracle.key \
+        --volume $<PATH_TO_PLUGINS_FILE>:/autoracle/plugins-conf.yml \
+        --name oracle-server \
+        --rm \
+        ghcr.io/clearmatics/autonity-oracle:v0.1.1-a2d406f \
+        -oracle_key_file="/autoracle/oracle.key" \
```
Comment (Jason-Zhangxin-Chen) [inline content/docs/oracle/run-oracle/_index.md]: Please update the flags' name again.

File: content/docs/reference/api/aut/op-prot/_index.md [hunk 9]
```diff
@@ -702,12 +796,11 @@ aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D setSymbol
 {{< /tab >}}
 {{< /tabpane >}}
 
-
 #### Example
 
 {{< tabpane langEqualsHeader=true >}}
 {{< tab header="aut" >}}
-aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D setSymbols ["AUD/USD", "CAD/USD", "EUR/USD", "GBP/USD", "JPY/USD", "SEK/USD", "ATN/USD", "NTN/ATN"]
+aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D setSymbols ["AUD-USD", "CAD-USD", "EUR-USD", "GBP-USD", "JPY-USD", "SEK-USD", "ATN-USD", "NTN-ATN"]
```
Comment (Jason-Zhangxin-Chen) [inline content/docs/reference/api/aut/op-prot/_index.md]: Since NTN-USD is required for the protocol, thus it would be better to add it as well in this example .

File: content/docs/reference/cli/oracle/_index.md [hunk 10]
```diff
@@ -21,9 +32,10 @@ Run `autoracle --help` to view the options:
 
 | ORACLE SERVER OPTIONS: | Description | Default | Required? |
 |:--|:--|:--|:--|
```
Comment (Jason-Zhangxin-Chen) [inline content/docs/reference/cli/oracle/_index.md:24]: Please rename those flags by refering to the README at https://github.com/autonity/autonity-oracle/blob/ue-improvement/README.md

The `oracle` prefix was removed and the `_` seperator is replace by `.`, some flag was renamed as well.

**Review by Jason-Zhangxin-Chen — COMMENTED** [review]
Leave some comments regarding to the configuration of oracle server, most of them are about the renaming of flags. Please refer to the latest README of oracle server: https://github.com/autonity/autonity-oracle/blob/ue-improvement/README.md

File: content/docs/reference/api/aut/op-prot/_index.md (hunk 9 above)
Comment (jay-cousins) [reply content/docs/reference/api/aut/op-prot/_index.md]: Added:

```
aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D setSymbols ["AUD-USD", "CAD-USD", "EUR-USD", "GBP-USD", "JPY-USD", "SEK-USD", "ATN-USD", "NTN-USD", "NTN-ATN"]
```

Resolved in commit 7b5a773d9d7c2590fad714687b3a1fcdea79e704

File: content/docs/oracle/run-oracle/_index.md (hunk 8 above)
Comment (jay-cousins) [reply content/docs/oracle/run-oracle/_index.md]: Done, resolved in commit 7b5a773d9d7c2590fad714687b3a1fcdea79e704

File: content/docs/oracle/run-oracle/_index.md (hunk 7 above)
Comment (jay-cousins) [reply content/docs/oracle/run-oracle/_index.md]: Done, resolved in commit 7b5a773d9d7c2590fad714687b3a1fcdea79e704

File: content/docs/oracle/run-oracle/_index.md (hunk 6 above)
Comment (jay-cousins) [reply content/docs/oracle/run-oracle/_index.md]: Done, resolved in commit 7b5a773d9d7c2590fad714687b3a1fcdea79e704

File: content/docs/reference/cli/oracle/_index.md (hunk 10 above)
Comment (jay-cousins) [reply content/docs/reference/cli/oracle/_index.md:24]: Done, resolved in commit 7b5a773d9d7c2590fad714687b3a1fcdea79e704

**Comment by jay-cousins** [comment]
Thanks for your review @Jason-Zhangxin-Chen - edits made for all your comments in commit Done, resolved in commit 7b5a773d9d7c2590fad714687b3a1fcdea79e704.

File: content/docs/oracle/run-oracle/_index.md [hunk 11]
```diff
@@ -34,62 +35,142 @@ Transaction costs for submitting price report data on-chain _are_ refunded but t
...
     ``` bash
     autoracle \
-        -oracle_key_file="<KEYFILE>" \
-        -oracle_key_password="<PWD>" \
-        -oracle_autonity_ws_url="<WS_ADDRESS>"
-        -oracle_plugin_conf="<PLUGINS_CONFIG_FILE>/plugins-conf.yml" ;
+        -key.file="<ORACLE_KEYFILE>" \
+        -key.password="<PWD>" \
+        -ws="<WS_ADDRESS>"
+        -plugin.conf="<PLUGINS_CONFIG_FILE>/plugins-conf.yml" ;
```
Comment (aiman) [inline content/docs/oracle/run-oracle/_index.md]:
```suggestion
        -ws="<WS_ADDRESS>" \
        -plugin.conf="<PLUGINS_CONFIG_FILE>/plugins-conf.yml"
```

Comment (aiman) [reply content/docs/oracle/run-oracle/_index.md]: This was missing a backslash

**Review by Jason-Zhangxin-Chen — APPROVED** [review]
LGTM

File: content/docs/oracle/run-oracle/_index.md (hunk 11 above)
Comment (jay-cousins) [reply content/docs/oracle/run-oracle/_index.md]: Added in commit b0b2cbe42f886c6b6a065832b949404f4aa216ae
